gym_handeye/utils/utils.py

```diff
@@ -0,0 +1,47 @@
+import gym_handeye as he
+
+
+def get_all_possible_transitions(grid_size):
```

Comment:
Zbędne

Reply:
Ta funkcja jest potrzebna do tego, by później w pyalcs można było określić, jak dużo agent wie o środowisku (ile z możliwych przejść umie przewidzieć). Podobna funkcja jest w środowisku Maze. Do tego też potrzebowałam klasy MockHandEye - żeby można było ustawić stan początkowy środowiska, wykonać akcję, a następnie uzyskać stan końcowy.

Reply:
Tak - tylko że w Maze potrzebowaliśmy jej żeby obliczyć tzw. wiedzę klasyfikatora. Czy potrafi z każdego stanu pewnie przejść do kolejnego - tutaj podejrzewam nie ma czegoś takiego. Może są inne metryki specyficzne dla tego środowiska (w publikacjach), które moglibyśmy tutaj wspomoć obliczeniami

Reply:
Wg artykułu "Action-Planning in Anticipatory Classifier Systems" Butza i Stolzmanna w punkcie 3.4 ("Testing the knowledge") właśnie coś podobnego było używane, żeby sprawdzić, jak dużo klasyfikator wie. Poprawiłam trochę kod i wyniki liczbowe wychodzą takie same, jak w tym artykule w tabeli 4., więc jeśli to o to chodzi, to wydaje mi się, że ta część kodu może zostać, chociaż mogę się mylić.